config/routes.rb

```diff
@@ -1,10 +1,9 @@
 Rails.application.routes.draw do
-  
-   get 'personal_page/index'
-   get 'fun_page/index'
+  get 'api_page/index'
```

Comment:
the index action should be give to you by default when you use `resources`. You can have an `only` option that just says you want the index action.